.github/workflows/submit-learning-path-pr.yml

```diff
@@ -0,0 +1,117 @@
+name: 'Submit Learning Path PR'
```

Comment:
I'm not sure I see the value of having split workflows here. It's adding a lot of complexity (and potential security risks) without adding value now that the proposed issue step has been removed.

Let's consolidate the split workflows and actions into a single one instead. If we consolidate them, then the action can also be responsible for generating a report (pr body) instead of having a bash script do it which is more prone to script injections.